src/components/icons/ErrorIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { createIcon } from '@chakra-ui/react';
+
+export const ErrorIcon = createIcon({
+  displayName: 'Error',
+  path: (
+    <svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Not 100% sure but based on Chakra [documentation](https://chakra-ui.com/docs/components/icon/usage#using-the-createicon-function), you should not pass the surrounding svg, only path. You can pass ViewBox separately to the function

Reply:
True! I will make a cleanup on the rest of the icons as well . Thank you 🌸